src/xcode/ENA/ENA/Source/Workers/Store/Store.swift

```diff
@@ -181,6 +181,8 @@ protocol CoronaTestStoringLegacy {
 
 	var isSubmissionConsentGiven: Bool { get set }
 
+	var profile: RapidTestProfile? { get set }
```

Comment:
```suggestion
	var anitgenTestProfile: RapidTestProfile? { get set }
```

Maybe to make it more clear as there is so much stuff in the store.